### ISSUE TITLE
Feature Request: Named Repository in "Recent Repositories"

### ISSUE THREAD

**Issue by MunizErthal** [gitextensions/gitextensions#9802]
### Feature description

Add name / alias to a repository instead of seeing the path only. 

![image](https://user-images.githubusercontent.com/32694606/148551795-441061b9-4f22-4a88-91f2-a0f4d1d4b4f2.png)

Thanks!

### Environment

- GitExtensions version:
- GIT version: 
- OS version: 
- .NET version:

**Comment by MunizErthal** [comment]
It's exactly the situation you said.
But I don't want to depend on the final path
If he displays the .git\description it would be perfect and resolve the situation.

**Comment by gerhardol** [comment]
> Note: `.git\description` is common for all the worktrees, so I don't know if it's a problem.

For worktrees use the worktree specific file like `.git/worktrees/gitextensions_4/description`

> If he displays the .git\description it would be perfect and resolve the situation.

Then you have the functionality

**Comment by MunizErthal** [comment]
No, because the .git\description is not displayed in the recent repository, just path.

**Comment by gerhardol** [comment]
> No, because the .git\description is not displayed in the recent repository, just path.

OK, only in title
